spec/services/withdraw_offer_spec.rb

```diff
@@ -6,15 +6,24 @@
       application_choice = create(:application_choice, status: :offer)
 
       withdrawal_reason = 'We are so sorry...'
-      WithdrawOffer.new(application_choice: application_choice, offer_withdrawal_reason: withdrawal_reason).save
+      WithdrawOffer.new(
+        actor: create(:support_user),
```

Comment:
Should we have one test where we make `ProviderAuthorisation` blow up? (here and in the other places it's added)

Reply:
I've added some `NotAuthorised` expectations to `WithdrawOffer`, `ConfirmOfferConditions`, `ConditionsNotMet` and `ConfirmEnrolment` service specs.

Reply:
Nice, thank you!